src/depthcloud/DepthCloud.js

```diff
@@ -160,9 +163,9 @@ ROS3D.DepthCloud = function(options) {
     '    float z = -depth;',
     '    ',
     '    pos = vec4(',
-    '      ( position.x / width - 0.5 ) * z * (1000.0/focallength) * -1.0,',
-    '      ( position.y / height - 0.5 ) * z * (1000.0/focallength),',
-    '      (- z + zOffset / 1000.0) * 2.0,',
+    '      ( position.x / width - 0.5 ) * z * 0.5 * maxDepthPerTile * (1000.0/focallength) * -1.0,',
```

Comment:
It looks like it changes the default behavior with default value of maxDepthPerTile. Can you make the default behavior as before?

Reply:
As I mentioned in the PR description, when https://github.com/RobotWebTools/depthcloud_encoder/pull/5 was merged, this file hasn't been updated for the decoded depth cloud to accommodate for the reduced factor. This was actually a bug that places all points at twice the distance from their correct pose, because we are encoding at a singular/unit range (1.0 = 0 - 2m) and decoding at twice the range (2.0 = ~4m). Hence this PR halves the distance in each axis.

Reply:
There actually was a PR that never made it in, because of too many untested changes. The change in question is here: https://github.com/RobotWebTools/ros3djs/pull/106/files#diff-32279e45de1135eb1be4b5f7120f695aL156

Reply:
@viktorku oh I see, any thoughts on a more general solution? For instance do you think it may be possible to provide the min/max range values directly in the data from the depthcloud_encoder so that the resolution can be dynamically and automatically configured on the receiving side here in ros3djs? It seems our current approach is a pretty poor user experience to have to tune both sides manually, maybe we should go after a more general improvement after we get this bug fix through.

I'll give this a spin on my equipment this evening.

Reply:
@viktorku I see. I missed to read your comments. thanks for the explanation.
@sevenbitbyte maybe user can `rosparam get max_depth_per_tile` before creating `DepthCloud` object to get the parameter from server side. How about adding this step it in our [depthcloud example](https://github.com/RobotWebTools/ros3djs/blob/develop/examples/depthcloud.html)?

Reply:
@sevenbitbyte any updates? Otherwise, we move on.